website/templates/project/settings.mako

```diff
@@ -305,61 +309,69 @@
                     </div>
                     <div class="panel-body">
                         % if not node['institution']['name']:
-                            <div data-bind="visible: !loading()">
-                                <div data-bind="visible: error()">
-                                    <div class="help-block">
-                                        Could not load up available institutions. Please wait a few minutes and try again, or contact <a href="mailto:support@osf.io">support@osf.io</a> if the problem persists.
-                                    </div>
-                                </div>
-                                <div data-bind="visible: !error()">
-                                    <div data-bind="visible: availableInstitutions().length">
+                            %if 'admin' in user['permissions']:
```

Comment:
Your logic looks something like this:
```
if node has no associated institution:
    ...
   if user is an admin:
       ....
   else
       ....
else:
       .....
````  
But then you end up duplicating the same lines for when "user is an admin but has no affiliated institutions" and "user is not an admin".

If the node is not affiliated with an institution and not a reg, it looks like there are two possible things they can see.  I would break it into:

**If user is an admin on the node, and the user is affiliated with an institution show this:**
![screen shot 2016-04-19 at 4 55 14 pm](https://cloud.githubusercontent.com/assets/9755598/14657828/ec8a358a-065d-11e6-9013-6231143cd6f3.png)
**Otherwise show this:**
![screen shot 2016-04-19 at 4 47 24 pm](https://cloud.githubusercontent.com/assets/9755598/14657847/0f62200e-065e-11e6-808d-c38f52b43279.png)